### ISSUE TITLE
Getting two errors when running on expo web

### ISSUE THREAD

**Issue by amandeepmittal** [tamagui/tamagui-expo-example#3]
I've cloned the repository and was trying out the example without any modification. It works on `iOS` but when running opening Web from Expo Developer Tools (CLI) doesn't show anything on the page gives me the following error in browser console:

```js
getStylesAtomic.js:9 Uncaught TypeError: i18Style is not a function
    at generateStyle (getStylesAtomic.js:9)
    at getAtomicStyle (getStylesAtomic.js:46)
    at getStylesAtomic.js:31
    at Array.map (<anonymous>)
    at getStylesAtomic (getStylesAtomic.js:30)
    at addStylesUsingClassname (addStylesUsingClassname.js:35)
    at addStylesUsingClassname (addStylesUsingClassname.js:32)
    at createComponent.js:344
    at createTamagui.js:121
    at Set.forEach (<anonymous>)
    at createTamagui (createTamagui.js:121)
    at Module../tamagui.config.ts (tamagui.config.ts:7)
    at __webpack_require__ (bootstrap:789)
    at fn (bootstrap:100)
    at Module../App.tsx (log.js:59)
    at __webpack_require__ (bootstrap:789)
    at fn (bootstrap:100)
    at Module../node_modules/expo/AppEntry.js (AppEntry.js:1)
    at __webpack_require__ (bootstrap:789)
    at fn (bootstrap:100)
    at Object.1 (tamagui.config.ts:37)
    at __webpack_require__ (bootstrap:789)
    at bootstrap:856
    at bootstrap:856
VM124:2 Uncaught ReferenceError: process is not defined
    at Object.4043 (<anonymous>:2:13168)
    at r (<anonymous>:2:306599)
    at Object.8048 (<anonymous>:2:9496)
    at r (<anonymous>:2:306599)
    at Object.8641 (<anonymous>:2:1379)
    at r (<anonymous>:2:306599)
    at <anonymous>:2:315627
    at <anonymous>:2:324225
    at <anonymous>:2:324229
    at HTMLIFrameElement.e.onload (index.js:1)
```

Can you please provide more information on how to fix this or if there is any other configuration required to use this example on Web?

**Comment by natew** [comment]
Should be fixed with 86da379e3d922a732a9642d7ebb64b1cce714751

Theme's aren't inserting variables, I could do it manually with `Tamagui.getCSS()` but I'd like to add a prop to `<Tamagui.Provider />` like `autoInsertThemeCSS` so it's not a weird process. It's separate for now since stuff like Next.js needs it to insert at SSR, but for non-SSR it's easier like so.

**Comment by amandeepmittal** [comment]
Thank you so much @natew for looking into it and releasing the fix 🙏
